content/source/docs/cloud/users-teams-organizations/single-sign-on/custom-saml.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: "cloud"
+page_title: "SAML - Single Sign-on - Terraform Cloud and Terraform Enterprise"
+---
+
+-> **Note:** Single sign-on is a paid feature, available as part of the **Business** upgrade package. [Learn more about Terraform Cloud pricing here](https://www.hashicorp.com/products/terraform/pricing/).
+
+# Single Sign-on: SAML
+
+The SAML SSO integration currently supports the following features of SAML 2.0:
+
+- Service Provider (SP)-initiated SSO
+- Identity Provider (IdP)-initiaited SSO
+- Just-in-Time Provisioning
+
+The SAML SSO integration can be configured by providing a metadata URL or manually with the Single Sign-on URL (ACS URL), Entity ID or Issuer URL, and X.509 Certificate.
```

Comment:
My understanding is that ACS and SSO are two different URLs. (And I thought ACS specific to SP.) Pinging just in case.

Reply:
Good catch. Not sure why I added this.